### ISSUE TITLE
JDK8 Test failure: org.eclipse.jetty.io.SelectChannelEndPointTest.testIdle

### ISSUE THREAD

**Issue by joakime** [eclipse/jetty.project#2240]
`jetty-9.4.x` branch.
Linux build.
Docker container build.

```
Error
Expected: a value greater than <1000L>
but: <999L> was less than <1000L>

Stacktrace
java.lang.AssertionError:
Expected: a value greater than <1000L>
but: <999L> was less than <1000L>
at org.eclipse.jetty.io.SelectChannelEndPointTest.testIdle(SelectChannelEndPointTest.java:467)
```

**Comment by joakime** [comment]
Cause/Fix might be similar to #2239

**Comment by joakime** [comment]
Closing as duplicate of #2226
